lib/cisco_node_utils/bgp_neighbor.rb

```diff
@@ -469,6 +470,8 @@ def mode_symbol_to_cli(symbol)
         'passive-only'
       when :both
         'both'
+      when :none
+        ''
```

Comment:
Is `:both` still relevant? The getters and setters can't use it so can it be removed altogether?